langchain4j/src/main/java/dev/langchain4j/chain/ConversationalRetrievalChain.java

```diff
@@ -36,15 +36,19 @@ public class ConversationalRetrievalChain implements Chain<String, String> {
     private final PromptTemplate promptTemplate;
     private final Retriever<TextSegment> retriever;
 
+    private final List<String> metadata;
+
     @Builder
     public ConversationalRetrievalChain(ChatLanguageModel chatLanguageModel,
                                         ChatMemory chatMemory,
                                         PromptTemplate promptTemplate,
-                                        Retriever<TextSegment> retriever) {
+                                        Retriever<TextSegment> retriever,
+                                        List<String> metadata) {
```

Comment:
I would name it a bit more specific, something like `metadataKeysToInclude` or `includedMetadataKeys`